base/doc/ltnews35.tex

```diff
@@ -178,9 +178,9 @@ \section{Document metadata interface}
 means that existing documents are in no way affected by the work that
 is carried out and is in a wider alpha or beta test phase.
 
-\emph{either add info about some already existing keys or add a
-  reference to the place the the right documentation about this is
-  stored (to be determined).}
+Documentation about the new command and
+already existing keys are in \file{l3meta.pdf} and \file{documentmetadata-support.pdf}
+and also in the documentation of the \pkg{pdfmanagement-testphase} package.
```

Comment:
maybe you are right and the support files should all be called latex-lab-xxx.ltx and the files supporting testphases testphase-yyy.ltx, e.g., instead of `pdfmangement-testphase` `latex-lab-pdfmanagement`. But I guess it doesn't really matter that much because most of this is essentially internal.